### ISSUE TITLE
create a project with abp cli by referencing locall but I am getting error

### ISSUE THREAD

**Issue by eemogz** [abpframework/abp#19838]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

I am trying to create a project with abp cli by referencing locall but I am getting the attached errors. The cli command I am using is “abp new MyAbpProject -t app --local-framework-ref --abp-path ‘C:\Users\Oguzhan\Desktop\abp_git_clone\framework’ ”
![local-path](https://github.com/abpframework/abp/assets/86205755/ebc771e8-e2ba-403e-bfda-ad59b55fcd0e)

### Version

abp clı 8.1.2

### User Interface

MVC

### Database Provider

EF Core (Default)

### Tiered or separate authentication server

None (Default)

### Operation System

Windows (Default)

**Comment by maliming** [comment]
hi

If the projects exist. You can try to build it using the `dotnet build` command.

![image](https://github.com/abpframework/abp/assets/6908465/444ccb22-bd7f-4d2f-92e3-3f3c6eea360c)

**Comment by eemogz** [comment]
I tried but I can't fix it, it gives an error in the IIdentityDbContext class, it asks me to do something that it normally doesn't want to do, language settings explode in the domain layer. Please tell me step by step how to pull and use mvc pages with abp starter kit repository classes and application services. The steps given in the document are incomplete, hard to follow.


![Ekran_goruntusu_2024-05-10_180707](https://github.com/abpframework/abp/assets/86205755/aa8d54b1-4e25-4a33-b980-102f9181527b)

**Comment by maliming** [comment]
Ensure the `abp_git_clone` repostory is on `rel-8.1` branch when you use `8.1.2` template project.

**Comment by eemogz** [comment]
I am using the latest version of abp cli and when I pulled the project from git repo https://github.com/abpframework/abp.git I pulled it with vs studio via this link.

@maliming
